src/components/SummaryList.scss

```diff
@@ -0,0 +1,42 @@
+.SummaryList-header {
+  color: $base-font-color;
+  .SummaryList-header-description {
+    color: $base-font-color;
+    float: left;
+  }
+  .SummaryList-header-value {
+    color: $base-font-color;
+    float: right;
+    clear: right;
+  }
+}
+
+.SummaryList-row {
+  border-top: $border-width-large solid $ts-grey;
+  padding: $su-default 0;
+  list-style-type: none;
+  li {
+    display: flex;
+    width: 100%;
+    height: 30px;
+    align-items: center;
+    .SummaryList-item-value {
```

Comment:
Nit: Should be camel case `SummaryList-itemValue` and doesn't need to be nested under parent.

Reply:
I took the classes off of these and just put the styles on the spans within the SummaryList-item above